code/app/commodity/server.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/brahma-adshonor/microservice/code/app/commodity/model"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+var (
+	db  *gorm.DB
+	err error
+)
+
+func hello(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprintf(w, "hello\n")
+}
+func getProduct(w http.ResponseWriter, r *http.Request) {
+	var request ProductRequst
+	var response ProductResponse
+
+	fmt.Printf("http body is : %v\n", r.Body)
```

Comment:
should replace `fmt` with `log` ？see at https://pkg.go.dev/log

Reply:
maybe next stage

Reply:
next time, could import 'zap' directly.

also could have analyses